README.md

```diff
@@ -0,0 +1,51 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project 1: Positioning and Floating Elements
+
+> Learning how to position and float elements by cloning an article on New York Times
+
+
+## Built With
+
+- Languages - HTML, CSS
+- Frameworks - boostraps
+- Technology - Linters
```

Comment:
add the technologies used for this properly: Frameworks: N/A, Technology: linters, git, github

Reply:
Updated. Thanks.